exercises/APE5-6/APE05-06E1b.tex

```diff
@@ -0,0 +1,58 @@
+We can proceed exactly as for the case a. We can first analyze the four pure strategies and we already find two Nash eqquilibria :
+
+- $\mathbf{\{x_1\}}\times\mathbf{\{x_2\}}$. Indeed step 3 implies :
+\begin{align*}
+    3 =w_1&\ge u_1(x_1,x_2)=2 \\
+    7=w_2&\ge u_2(x_1,y_2)=6 
+\end{align*}
+And so all the step are respected (note that step 1 and 2 are trivial for the pure strategies) 
+
+- $\mathbf{\{y_1\}}\times\mathbf{\{y_2\}}$. Indeed step 3 implies :
+\begin{align*}
+    7 =w_1&\ge u_1(x_1,y_2)=6 \\
+    3=w_2&\ge u_2(y_1,x_2)=2
+\end{align*}
+Again all the step are respected.
+
+
+\begin{itemize}
+  \item[$\bullet$] Randomized vs pure strategy supports
+\end{itemize}
+
+One can easily develop Payoff equation (step 2) as in Exercise 1 and see that all support does not lead to a Nash equilibrium.
```

Comment:
Avoid using "easily", if someone is reading the solution it might be that it wasn't easy for him and he might get offended :)